src/sys/socket/addr.rs

```diff
@@ -23,15 +23,15 @@ use ::sys::socket::addr::sys_control::SysControlAddr;
 #[repr(i32)]
 #[derive(Copy, Clone, PartialEq, Eq, Debug, Hash)]
 pub enum AddressFamily {
-    Unix = consts::AF_UNIX,
-    Inet = consts::AF_INET,
-    Inet6 = consts::AF_INET6,
+    Unix = libc::AF_UNIX,
+    Inet = libc::AF_INET,
+    Inet6 = libc::AF_INET6,
     #[cfg(any(target_os = "linux", target_os = "android"))]
-    Netlink = consts::AF_NETLINK,
+    Netlink = libc::AF_NETLINK,
     #[cfg(any(target_os = "linux", target_os = "android"))]
-    Packet = consts::AF_PACKET,
+    Packet = libc::AF_PACKET,
     #[cfg(any(target_os = "macos", target_os = "ios"))]
-    System = consts::AF_SYSTEM,
+    System = libc::AF_SYSTEM,
```

Comment:
Ok, I'll go through the constants defined in `libc` and try to merge all them in here.